db/migrate/20200427175819_set_read_only_flag.rb

```diff
@@ -0,0 +1,12 @@
+class SetReadOnlyFlag < ActiveRecord::Migration[5.1]
+  class Dialog < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    include ActiveRecord::IdRegions
+  end
+
+  def up
+    say_with_time("Updating Dialogs") do
+      Dialog.in_my_region.all.each { |dialog| dialog.label == 'Transform VM' ? dialog.update(:system => true) : dialog.update(:system => false) }
```

Comment:
What do you think of this?  It prevents loading every record and enumerating through them one-by-one.
```suggestion
      Dialog.in_my_region.where(:label => 'Transform VM').update_all(:system => true)
      Dialog.in_my_region.where.not(:label => 'Transform VM').update_all(:system => false)
```

Reply:
👍 thanks